test/Microsoft.AspNet.FileProviders.Tests/EmbeddedFileProviderTests.cs

```diff
@@ -14,7 +14,7 @@ public class EmbeddedFileProviderTests
         [Fact]
         public void When_GetFileInfo_and_resource_does_not_exist_then_should_not_get_file_info()
         {
-            var provider = new EmbeddedFileProvider(this.GetType().Assembly, "");
+            var provider = new EmbeddedFileProvider(this.GetType().Assembly, "Microsoft.AspNet.FileProviders.Tests");
```

Comment:
Pull the namespace into a variable - 
`private static readonly ResourceBaseNamespace = typeof(EmbeddedFileProviderTests).Namespace;`